startup/db/meetupDb.js

```diff
@@ -1,18 +1,16 @@
-const today = new Date();
-
 const meetupDb = [
   {
     id: 0,
     title: 'coders brings live',
     location: 'Abuja',
-    happeningOn: new Date(today.setHours(today.getHours() + 2000)),
+    happeningOn: '12-04-2016',
```

Comment:
Why did you hard code this value?

Reply:
Initially, I wasn't implementing the get upcoming meetup routes well, so I used that to set upcoming dates but, I have changed the setup to the one on line 6